lang/c/c.talon

```diff
@@ -145,14 +145,15 @@ push brackets:
     insert("{variable} {letter} ")
 
 # Ex. int * testFunction
-fun <user.function> <phrase>:
-    insert("{function} ")
-    insert(user.formatted_text(phrase, "PRIVATE_CAMEL_CASE,NO_SPACES"))
-    insert("()")
-    edit.left()
+# TODO: these clearly don't want to use function_key, so what do they want to use?
+# fun <user.function> <phrase>:
```

Comment:
Looks like user.function here is a copy/paste error. Should probably just be fun <user.text>?

This might be out of scope, but we may want to take a moment to update this to use the newer .py actions e.g. `code_private_function` as in python, csharp and r.